### ISSUE TITLE
Gait Manager

### ISSUE THREAD

**Issue by DavidMansolino** [darwinop/webots-cross-compilation#45]
@fabienrohrer 
I have tested the Gait Manager:
X_MOVE_AMPLITUDE = step length (front or rear) [unit = 0.5cm each step (right and left)]
Z_MOVE_AMPLITUDE = height of the foot during the step.
A_MOVE_AMPLITUDE = rotation on itself [ unit ~ 0.7deg each step (right and left)]
BALANCE_ENABLE = enable the balance, the balance use only gyroscope (contrary to what I said to you yesterday), It mainly avoid to fall forward or backward, but concerning the lateral stability, I haven't notice any difference.
Y_MOVE_AMPLITUDE = a kind of side step, but do not work very very well (very small step, parasite rotation and X movements), work better with biggest value (20-25), the limit of 15 currently fixed in the manager is to low.
A_MOVE_AIM_ON = has no effect if A_MOVE_AMPLITUDE=0, otherwise it change the direction of rotation, and the robot seems to make a circle (the radius is modified with A_MOVE_AMPLITUDE).

**Comment by fabienrohrer** [comment]
To be sure: did you test that on the real robot?

**Comment by DavidMansolino** [comment]
Yes, all these results are from the real robot and using the Robotis Framework only.

**Comment by fabienrohrer** [comment]
Maybe y_move_amplitude is related with a_move_aim_on (?)

**Comment by DavidMansolino** [comment]
No, I just test and I found no correlation between A_MOVE_AIM_ON and Y_MOVE_AMPLITUDE.
I think A_MOVE_AIM_ON is only related with A_MOVE_AMPLITUDE (I have inspected the walking algorithm and it seems to be in agreement with my first idea).

But I found something else interesting, the sidestep is much more better when balancing is enable (maybe it is why the sidestep currently do not work at all in Webots) and when the value is around 20.

**Comment by fabienrohrer** [comment]
Thank you for these interesting results
